submariner-operator/templates/submariner.yaml

```
@@ -23,6 +23,32 @@ spec:
   natEnabled: {{ .Values.submariner.natEnabled }}
   repository: {{ .Values.submariner.images.repository }}
   version: {{ .Values.submariner.images.tag }}
+{{- with .Values.images }}
+{{- if . }}
+  imageOverrides:
+    {{- if index . "submariner-operator" }}
+    submariner-operator: {{ index . "submariner-operator" }}
+    {{- end}}
```

For stylistic consistency (likewise for other `{{ end }}`s):

```suggestion
    {{- end }}
```